packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -30,6 +30,18 @@ const publicUrl = '';
 // Get environment variables to inject into our app.
 const env = getClientEnvironment(publicUrl);
 
+let extraSassLoaders = [];
+if (process.env.PATHS_TO_SASS_RESOURCES_TO_INJECT) {
+  extraSassLoaders.push({
+    loader: require.resolve('sass-resources-loader'),
+    options: {
+      resources: process.env.PATHS_TO_SASS_RESOURCES_TO_INJECT
```

Comment:
As agreed:  SASS_RESOURCES_TO_INJECT will be better naming

Reply:
What about `SASS_PATHS_RESOLVER`?

Reply:
But a Resolver it should take an action responsible to resolve something, but it is not resolving anything because the path is already resolved. :)

Reply:
`SASS_PATHS_INJECTOR`?